src/main/java/net/pms/dlna/DVDISOTitle.java

```diff
@@ -335,7 +380,120 @@ public DLNAThumbnailInputStream getThumbnailInputStream() throws IOException {
 		} else if (getMedia() != null && getMedia().getThumb() != null) {
 			return getMedia().getThumbnailInputStream();
 		} else {
-			return getGenericThumbnailInputStream("images/thumbnail-disc.png");
+			return getGenericThumbnailInputStream(null);
 		}
 	}
+
+	protected DLNAMediaAudio parseMEncoderAudioStream(String line) {
+		if (isBlank(line)) {
+			return null;
+		}
+		Matcher matcher = AUDIO_STREAM_PATTERN.matcher(line);
+		if (matcher.find()) {
+			DLNAMediaAudio audio = new DLNAMediaAudio();
+			try {
+				audio.setTrack(Integer.parseInt(matcher.group("StreamNumber")));
+			} catch (NumberFormatException e) {
+				LOGGER.error(
+					"Could not parse audio stream number \"{}\": {}",
+					matcher.group("StreamNumber"),
+					e.getMessage()
+				);
+				LOGGER.trace("", e);
+			}
+
+			audio.setCodecA(MPlayerDvdAudioStreamTypes.typeOf(matcher.group("Codec")).getFormatConfigurationCode());
+			audio.getAudioProperties().setNumberOfChannels(
+				MPlayerDvdAudioStreamChannels.typeOf(matcher.group("Channels")).getNumberOfChannels()
+			);
+			String languageCode = Iso639.getISOCode(matcher.group("Language"));
+			audio.setLang(isBlank(languageCode) ? DLNAMediaLang.UND : languageCode);
+			try {
+				audio.setId(Integer.parseInt(matcher.group("AID")));
+			} catch (NumberFormatException e) {
+				LOGGER.error("Could not parse audio id \"{}\": {}", matcher.group("AID"), e.getMessage());
+				LOGGER.trace("", e);
+			}
+			if (audio.getId() >= MPlayerDvdAudioStreamTypes.FIRST_LPCM_AID) {
+				if (!FormatConfiguration.LPCM.equals(audio.getCodecA())) {
+					LOGGER.warn(
+						"Unexpected error parsing DVD audio stream codec. AID dictates LPCM while codec is {}",
+						audio.getId(),
+						audio.getCodecA()
+					);
+				}
+			} else if (audio.getId() >= MPlayerDvdAudioStreamTypes.FIRST_DTS_AID) {
+				if (!FormatConfiguration.DTS.equals(audio.getCodecA())) {
+					LOGGER.warn(
+						"Unexpected error parsing DVD audio stream codec. AID dictates DTS while codec is {}",
+						audio.getId(),
+						audio.getCodecA()
+					);
+				}
+			} else if (audio.getId() >= MPlayerDvdAudioStreamTypes.FIRST_AC3_AID) {
+				if (!FormatConfiguration.AC3.equals(audio.getCodecA())) {
+					LOGGER.warn(
+						"Unexpected error parsing DVD audio stream codec. AID dictates AC3 while codec is {}",
+						audio.getId(),
+						audio.getCodecA()
+					);
+				}
+			} else if (!FormatConfiguration.MP2.equals(audio.getCodecA())) {
+				LOGGER.warn(
+					"Unexpected error parsing DVD audio stream codec. AID dictates MP2 while codec is {}",
+					audio.getId(),
+					audio.getCodecA()
+				);
+			}
+			/*
+			 * MPlayer doesn't give information about bit rate, sampling rate
```

Comment:
If i believe Google, it's not wrong:
bitrate  ==> 110 millions
bit rate ==> 13.4 millions

https://techterms.com/definition/bitrate  (My english is more bad, but my link is better LOL)